mocks/handlers/edit-invoice-handlers.ts

```diff
@@ -0,0 +1,85 @@
+import { HttpResponse, http } from 'msw';
+
+export const EDIT_INVOICE_HANDLERS = [
```

Comment:
This is fine for now, but you usually want to keep Edit / Get / Delete handlers inside one file, to be able to handle interactions.